src/app/modules/card/card.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'ukm-std-card',
```

Comment:
- file name should match the selector (std-card.component.ts)
- I think don't use "std" since it doesn't provide any meaningful information (ambiguous), try to use more informative name

why don't we just use material card directly? https://material.angular.io/components/card/overview
what makes this different from material card?